management/virtualmachine/client.go

```diff
@@ -648,8 +648,8 @@ func (self VirtualMachineClient) createServiceCertDeploymentConf(certPath string
 	return certConfig, nil
 }
 
-func (self VirtualMachineClient) createSshConfig(certPath, userName string) (SSH, error) {
-	sshConfig := SSH{}
+func (self VirtualMachineClient) createSshConfig(certPath, userName string) (*SSH, error) {
```

Comment:
That would need an extra variable at the call site [1]. This code is being deleted in #65 , so I don't really care. LMK if you still want me to make the change.

[1] https://github.com/paulmey/azure-sdk-for-go/blob/virtualmachine-cleanup/management/virtualmachine/client.go#L607-607